lectures/unpleasant.md

```diff
@@ -152,15 +154,15 @@ running monetary and fiscal policies.
 
 Here, by **fiscal policy** we mean the collection of actions that determine a sequence of net-of-interest government deficits $\{g_t\}_{t=0}^\infty$ that must be financed by issuing to the public  either money or interest bearing bonds.
 
-By **monetary policy** or **debt-management polcy**, we  mean the collection of actions that determine how the government divides its  portolio of debts to the public  between interest-bearing parts (government bonds) and non-interest-bearing parts (money).
+By **monetary policy** or *debt-management policy*, we  mean the collection of actions that determine how the government divides its  portolio of debts to the public  between interest-bearing parts (government bonds) and non-interest-bearing parts (money).
```

Comment:
I think **debt-management policy** is OK here, since it's a definition.

Reply:
thanks @jstac reading your comments as well to learn what to look out for. Thanks for this reminder re: definitions. 

https://manual.quantecon.org/styleguide/writing.html#emphasis-and-definitions